### PR TITLE
Add quote semantic search demo

### PR DESCRIPTION
**Pull request by jdogmcsteezy**
I did not update the global README since that is meant to funnel people to the prism image search demo

### CODE REVIEW

File: quote-semantic-search/quote-search/static/components/home.vue.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const Home = {
+    template: `
+    <div>
+        <h1 class="title">Proximus Image search</h1>
```
Comment (hev) [inline quote-semantic-search/quote-search/static/components/home.vue.js:4]:
```suggestion
        <h1 class="title">Proximus Semantic Text search</h1>
```

File: quote-semantic-search/quote-search/static/components/home.vue.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        <div class="p-3 align-items-center justify-content-center">
+        <p>
+          <div class="d-flex flex-column flex-lg-row align-items-md-stretch justify-content-md-center gap-3 mb-4">
+                <img class="proximus img-responsive center-block" src="/images/proximus.png" alt="Proximus"/>
+          </div>
+        </p>
+
+        <p>
+        This demo application provides semantic search for a set of images
```
Comment (hev) [inline quote-semantic-search/quote-search/static/components/home.vue.js:13]: Need to update this description.

**Review by hev — CHANGES_REQUESTED** [review]

File: quote-semantic-search/quote-search/static/components/home.vue.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        <div class="p-3 align-items-center justify-content-center">
+        <p>
+          <div class="d-flex flex-column flex-lg-row align-items-md-stretch justify-content-md-center gap-3 mb-4">
+                <img class="proximus img-responsive center-block" src="/images/proximus.png" alt="Proximus"/>
+          </div>
+        </p>
+
+        <p>
+        This demo application provides semantic search for a set of quotes
+        by indexing them using the <a class="link-dark" href="https://huggingface.co/sentence-transformers/clip-ViT-B-32-multilingual-v1">CLIP</a>
```
Comment (hev) [inline quote-semantic-search/quote-search/static/components/home.vue.js:14]: Model still referenced as CLIP here.

**Review by hev — APPROVED** [review]

File: prism-image-search/prism/routes.py [hunk 4]
```diff
@@ -36,7 +36,7 @@ def search():
         time_taken = time.time() - start
         return format_results(results, time_taken)
     else:
-        return "No image uploaded", 400
+        return "No test uploaded", 400
```
Comment (jdogmcsteezy) [inline prism-image-search/prism/routes.py:39]:
```suggestion
        return "No text uploaded", 400
```

File: quote-semantic-search/quote-search/static/components/home.vue.js (hunk 3 above)
Comment (hev) [reply quote-semantic-search/quote-search/static/components/home.vue.js:14]: Meant to request changes here.

**Review by hev — CHANGES_REQUESTED** [review]

File: quote-semantic-search/quote-search/static/components/home.vue.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        model created by OpenAI. This model generates vectors with semantic meaning 
+        from each quote and stores them as vector embeddings in Aerospike. When a user
```
Comment (jdogmcsteezy) [inline quote-semantic-search/quote-search/static/components/home.vue.js:16]:
```suggestion
        by indexing them using the <a class="link-dark" href="https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2">MiniLM</a>
        model. This model generates vectors with semantic meaning 
        from each quote and stores them as vector embeddings in Aerospike. When a user
```

File: quote-semantic-search/quote-search/static/components/home.vue.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+        <h1 class="title">Proximus Quote search</h1>
+        <div class="p-3 align-items-center justify-content-center">
+        <p>
+          <div class="d-flex flex-column flex-lg-row align-items-md-stretch justify-content-md-center gap-3 mb-4">
+                <img class="proximus img-responsive center-block" src="/images/proximus.png" alt="Proximus"/>
+          </div>
+        </p>
+
+        <p>
+        This demo application provides semantic search for a set of quotes
```
Comment (hev) [inline quote-semantic-search/quote-search/static/components/home.vue.js:13]: I think we may also want to link where the quotes came from.

**Review by hev — CHANGES_REQUESTED** [review]

**Review by hev — APPROVED** [review]

**Pull request merged**